core/bid.py

```diff
@@ -112,7 +112,7 @@ def bid(q, api, playerList, settings):
                     for item in api.searchAuctions('player', defId=defId, max_price=bidDetails[defId]['maxBid']-subtract, start=0, page_size=50):
                         # player safety checks for every possible bid
                         # Let's look at last 5 minutes for now and bid on 5 players max
-                        if item['expires'] > 600 or bidon >= 5 or listed >= settings['maxPlayer'] or api.credits < settings['minCredits']:
+                        if item['expires'] > 300 or bidon >= 5 or listed >= settings['maxPlayer'] or api.credits < settings['minCredits']:
```

Comment:
I think 5 minutes is fine since this doesn't affect much the probabilities of winning a bid: if a price will go over the threshold in those 5 minutes then there is nothing you can do about it.

The only difference could be for BIN sniping ut I am not sure on how to improve this (we should come up with something that goes over multiple pages of cards automatically).